util/settings.go

```diff
@@ -27,6 +27,8 @@ var MethodAPIKey = "apikey"
 type Settings struct {
 	// The JSON credentials content downloaded from Google Cloud Console.
 	CredentialsJSON string
+	// The authentication method should be used.
```

Comment:
// The authentication method that should be used. 

Again, consider naming this "AuthType" since that's what the cli parameter uses : )